en/drive_client/drive_client_for_linux.md

```diff
@@ -4,7 +4,7 @@ You can find supported OS versions on <https://cloud.seatable.io/dtable/external
 
 ## Installing on Debian/Ubuntu
 
-To install the client, first add the signing key:
+To install the client, first add the signing key which has been updated:
```

Comment:
不应该这样写，因为这个 key 不是每次都被更新了。应该说，If apt-get reports following error: "The following signatures couldn't be verified because the public key is not available", please update the key for seafile repository.